probe-dictionary/explorer.css

```diff
@@ -14,6 +14,7 @@ body {
     padding: 7.5px 8px;
     line-height: 1.5;
     color:rgba(255,255,255,.75);
+    font-weight: 700;
```

Comment:
We just use `font-weight: bold;` for `.nav-item`, let's use the same for this button.